.github/workflows/on-push-to-main-branch.yml

```diff
@@ -18,6 +18,9 @@ jobs:
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
+    - name: Get release version
+      id: tag
+      uses: divyansh-gupta/action-get-tag@727a6f0a561be04e09013531e73a3983a65e3479
```

Comment:
We probably want to use the `git describe` VERSION here instead, I'm not sure what the tag here will return.

Reply:
Aren't we trying to test the injection of this so that on release it works? Maybe I'm mistaken

Reply:
Yes, but we don't need to test that the tag action is working, we know that works - we need to test that injection is working. The value being injected is not really relevant. For test ECR images I think we should emit the git version, not the release tag, or our metrics and data will be messed up between production/testing.

Reply:
Oh I see, let me change those back. Sorry about the miscommunication.